docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md

```diff
@@ -40,6 +40,12 @@ This policy is a relaxed version of the `restricted-noroot` policy, with almost
 
 This policy is equivalent to running Kubernetes with the PSP controller disabled. It has no restrictions on what pods can be deployed into a cluster or project.
 
+:::note important
```

Comment:
```suggestion
:::important
```

Reply:
While `:::important` works, it's an undocumented/legacy admonition and may break in the future. The original syntax here is correct. 

^@jiaqiluo

Reply:
Good to know. 
I reverted it to the original.